server/src/lib/model/user-client.ts

```diff
@@ -65,6 +65,39 @@ async function updateLocales(
   }
 }
 
+async function updateDemographics(
+  clientId: string,
+  age?: string,
+  sex?: string
+) {
+  const [[{ id: ageId }]] = await db.query(
+    `
+    SELECT id
+      FROM ages
+      WHERE age = ?
+  `,
+    [age]
+  );
+
+  const [[{ id: sexId }]] = await db.query(
+    `
+    SELECT id
+      FROM sexes
+      WHERE sex = ?
+  `,
+    [sex]
+  );
```

Comment:
This is awesome feedback, and may have caught a bug early. I'm going to look at this tomorrow with fresh eyes; since `age` and `sex` aren't necessarily sent up with every profile update, I want to make sure we're not eg. wiping someone's demographic data every time they toggle their `visible` setting.